modules/programs/mu.nix

```diff
@@ -31,6 +31,15 @@ in {
 
       package = mkPackageOption pkgs "mu" { };
 
+      home = mkOption {
+        type = with types; nullOr str;
+        default = null;
```

Comment:
Maybe the default should be `"${config.xdg.cacheHome}/mu"` and `--muhome` always used? Note, if we do this then also need to add `defaultText` to avoid making documentation depend on the configuration.

Reply:
Since the code already uses `dbLocation = config.xdg.cacheHome + "/mu"` we effectively already specify a default value. So I would suggest using something like

``` nix
type = types.path;
default = config.xdg.cacheHome + "/mu";
defaultText = literalExpression ''config.xdg.cacheHome + "/mu"'';
```

in the option

and `--muhome ${escapeShellArg cfg.home}` in the command.

I assume it is unlikely that mu changes the default directory again, if they do then we'll have to guard it behind a `stateVersion` to allow a sensible upgrade path.